ptbcontrib/username_to_chat_api/README.md

```diff
@@ -0,0 +1,29 @@
+# Wrapper for the [usernameToChatAPI](https://github.com/Poolitzer/usernameToChatAPI)
+
+This provides a wrapper for the [usernameToChatAPI](https://github.com/Poolitzer/usernameToChatAPI). The wrapper returns a Chat object like getChat would, and takes care of the (request) logic in the background.
```

Comment:
```suggestion
This provides a wrapper for the [usernameToChatAPI](https://github.com/Poolitzer/usernameToChatAPI). The wrapper returns a `telegram.Chat` object like `telegram.Bot.get_chat` would, and takes care of the (request) logic in the background.
```

I suggest to also make it clear that this requires a userbot in the background.

Reply:
hm? running the API you mean?

only if you actually run it, so it wouldn't make sense here. If you use an existing instance you personally don't need a userbot

Reply:
sure, but I have a feeling that people will get confused & think that it's actually possible with the bot API and then we have to explain why there is `usernameToChatAPI`

Reply:
Aha.